src/api/api_pass.cc

```diff
@@ -126,7 +126,7 @@ REGISTER_PASS1(ConvertSSA);
 REGISTER_PASS1(VerifySSA);
 REGISTER_PASS1(RewriteUnsafeSelect);
 REGISTER_PASS4(Inline);
-REGISTER_PASS3(StorageFlatten);
+REGISTER_PASS4(StorageFlatten);
```

Comment:
The new parameter of StorageFlatten may be made optional, defaulting to `false`. In this case the tests calling StorageFlatten don't have to be changed. (On the other hand, making `true` the default may be a better decision in the long run, I'm not sure)

Reply:
@sgrechanik-h thanks for review. Make sense to me to make new parameter as default.
I've executed tests locally - tests/scripts/task_python_unittest.sh and tests/scripts/task_python_topi.sh with bound checkers instrumentation by default, to verify the patch. The problem is that I'm not able to reproduce the stable tests suite same, as on remote CI, some tests fails even on master branch, and in this case I didn't find any new fails comparison to master branch, but it seems to me there are could be some tests which check the added attributes and they could fails, because if we set instrumentation to true every loads/stores instructions will mark with theirs bounds by attributes.
Thanks.